mucgpt-frontend/src/components/QuestionInput/QuestionInput.module.css

```
@@ -88,6 +96,33 @@
     min-width: 0;
 }
 
+.questionInputTextArea :global(textarea) {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)